README.md

```diff
@@ -8,18 +8,18 @@ This is the [html-report plugin](http://getgauge.io/documentation/user/current/p
 Install through Gauge
 ---------------------
 ```
-gauge --install html-report
+gauge install html-report
 ```
 
 * Installing specific version
 ```
-gauge --install html-report --plugin-version 2.1.0
+gauge install html-report --plugin-version 2.1.0
```

Comment:
`--plugin-version` is now `--version`
So this command should be
`gauge install html-report --version 2.1.0`